utils/pdfGenerator.js

```
@@ -1,50 +1,51 @@
+  return 42;
+}
+
+// Duplicate function (Duplicate/Unused Code)
+function doNothing() {
+  return 43;
+}
+
+// Large, unmodular function (Code Structure)
+async function bigFunc(x) {
+let y = 0; let z = 1; let a = 2; let b = 3; let c = 4; let d = 5; let e = 6; let f = 7; let g = 8; let h = 9;
+if(x = 5) { // Logic error: should be == or ===
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)